packages/core/react/src/__tests__/WalletProviderMobile-test.tsx

```diff
@@ -326,10 +326,14 @@ describe('WalletProvider when the environment is `MOBILE_WEB`', () => {
     describe('onError', () => {
         let onError: jest.Mock;
         let errorThrown: WalletError;
-        beforeEach(() => {
+        beforeEach(async () => {
             errorThrown = new WalletError('o no');
             onError = jest.fn();
             renderTest({ onError });
+            await act(async () => {
+                ref.current?.getWalletContextState().select(SolanaMobileWalletAdapterWalletName);
+                await Promise.resolve(); // Flush all promises in effects after calling `select()`.
+            });
```

Comment:
This change doesn't make sense to me. MWA shouldn't be deselected after each test, given the comment following this one)

Reply:
hey sorry, not sure I totally follow – I don't think there's any selecting involved, the test is just to make sure that when the adapter emits an error, it expects `onError` to be called. or was the point of the test something different?